frontend/src/components/utils/Balance.jsx

```diff
@@ -5,19 +5,23 @@ import { OverlayTrigger, Tooltip } from "react-bootstrap";
 
 const FRAC_DIGITS = 5;
 
-const Balance = ({ amount }) => {
+const Balance = ({ amount, label = null, className = "" }) => {
```

Comment:
It think it is better to leave className undefined if it was not defined

```suggestion
const Balance = ({ amount, label = null, className }) => {
```